app/schemas/users.js

```diff
@@ -0,0 +1,54 @@
+const { mailDomainError } = require('../errors');
+const { REGEX_EMAIL } = require('../constants');
+
+exports.createUser = {
+  firstName: {
+    in: ['body'],
+    exists: {
+      errorMessage: 'Empty firstname'
```

Comment:
You must define if all errors start with a capital letter or not, handle a standard